### ISSUE TITLE
Add Merlin to the installer

### ISSUE THREAD

**Issue by romildo** [protz/ocaml-installer#33]
[Merlin](http://the-lambda-church.github.io/merlin/) is an editor-independent tool to ease the development of programs in OCaml. It aims to provide features available in modern IDEs such as:
* _As you type_ error reporting
* Auto completion
* Extraction of type information
* Source browsing (_jump to definition_)

It may be very helpful to have Merlin installed with OCaml so that one can easily use Emacs+Tuareg+Merlin (or any other editor supported by Merlin) while developing on Windows. In particular it would make it easier for my students to set up OCaml for studying functional programming.

Please, add Merlin to ocaml-installer.

**Comment by msprotz** [comment]
"opam add merlin" works for "32-bit ocaml + cygwin-32 opam", but not for "64-bit ocaml + 64-bit opam"

**Comment by qdwang** [comment]
Is merlin still incompatible with 64bit version?

**Comment by msprotz** [comment]
At this stage we're waiting on proper OPAM support for Windows, which should hopefully happen early next year, so yeah, still not working, sadly.

**Comment by qdwang** [comment]
Is it possible that OPAM can natively support Windows(with out cygwin)? A lot of packages is based on *unix system features.  

[https://github.com/fdopen/opam-repository-mingw](https://github.com/fdopen/opam-repository-mingw)

I found that this repo‘s OPAM can successfully install a lot of packages like merlin, lwt and something else. [support list here](https://fdopen.github.io/opam-repository-mingw/packages/)

Don't know what modification did he do to the OPAM. May be a little help for you guys.

**Comment by msprotz** [comment]
Yes, we are aware of this repository. Until recently, Andreas hadn't released his patches to OPAM, so I didn't want to redistribute an arbitrary binary. Now he has released his patches to OPAM, but these are no longer maintained, and an OPAM on Windows effort is currently happening in Cambridge Labs, so our best bet is to wait a little bit longer and see what comes out of it.

Note that the installer's OPAM points to Andreas' repository by default.

**Comment by qdwang** [comment]
So that's it...hope that the Windows version can be released as soon as possible.
